yarn.lock

```diff
@@ -4430,9 +4430,9 @@ typescript-eslint-parser@11.0.0:
     lodash.unescape "4.0.1"
     semver "5.4.1"
 
-typescript@2.6.2:
-  version "2.6.2"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.6.2.tgz#3c5b6fd7f6de0914269027f03c0946758f7673a4"
+typescript@2.7.0-*:
```

Comment:
Not sure why there's a `*` 🤔, and I got the following diff by running `yarn` on this branch, weird.

![image](https://user-images.githubusercontent.com/8341033/34357309-f052c520-ea80-11e7-8b3a-3f8517663d24.png)

Reply:
Oops, forgot that the yarn interactive version selector thingy didn't persist the selection to yarn.lock. Fixed.